packages/base/src/keybindings.json

```diff
@@ -1,4 +1,14 @@
 [
+  {
+    "command": "jupytercad:undo",
+    "keys": ["Accel Z"],
+    "selector": ".jp-jupytercad-panel"
+  },
+  {
+    "command": "jupytercad:redo",
+    "keys": ["Accel Y"],
+    "selector": ".jp-jupytercad-panel"
```

Comment:
Should we do as in the other PR and add these keyboard shortcuts for the tree view?

At some point it will be repetitive to copy paste shortcuts for both the main view and the tree view, ideally we would define the shortcut only once for the entire app. But let's figure that out in a later-on PR. I don't want to block this PR for this.

Reply:
> At some point it will be repetitive to copy paste shortcuts for both the main view and the tree view, ideally we would define the shortcut only once for the entire app. But let's figure that out in a later-on PR. I don't want to block this PR for this.

One option is to have `body` as the selector and the other option is to wrap the components we want to have effects of key bindings in a container and then use that as a standard selector.

The latter is more fine grained but will take some time to implement and test effectively. Which I'll be more than happy to look into as well.

Reply:
If the body selector just works, I'm fine with it:

> I believe the command should behave properly and only execute the command on the currently active JupyterCad editor, if there is one.